drivers/SmartThings/samsung-audio/src/init.lua

```diff
@@ -213,11 +213,6 @@ local samsungAudio = Driver("samsung-audio", {
       [capabilities.audioVolume.commands.volumeDown.NAME] = handlers.handle_volume_down,
       [capabilities.audioVolume.commands.setVolume.NAME] = handlers.handle_set_volume,
     },
-    [capabilities.audioNotification.ID] = {
-      [capabilities.audioNotification.commands.playTrack.NAME] = handlers.handle_audio_notification,
-      [capabilities.audioNotification.commands.playTrackAndResume.NAME] = handlers.handle_audio_notification,
-      [capabilities.audioNotification.commands.playTrackAndRestore.NAME] = handlers.handle_audio_notification,
```

Comment:
We can remove the corresponding handler functions as well as they are no more needed. Do we want to keep them?

Reply:
In my opinion, no harm in keeping this if by chance we need handler in future.